js/adapt-contrib-slider.js

```diff
@@ -256,11 +256,16 @@ define(function(require) {
 
         onNumberSelected: function(event) {
             event.preventDefault();
-            if (this.model.get('_isComplete')) return;
-            var index = parseInt($(event.currentTarget).attr('data-id')) - 1;
+
+            if (this.model.get('_isComplete')) {
+              return;
+            }
+
+            var index = this.getIndexFromValue(parseInt($(event.currentTarget).attr('data-id')));
+            var $scaler = this.$('.slider-scaler');
             this.selectItem(index);
-            this.animateToPosition(this.mapIndexToPixels(index));
-            this.setAltText(index + 1);
+            this.animateToPosition(this.mapIndexToPixels(index, $scaler));
+            this.setAltText(index);
```

Comment:
yes, it's definitely getting it wrong because `getIndexFromValue` is returned a 0-based index of the markers... so it's even more inaccurate if you change the range to be something like 10 - 20 - clicking on 15 will give you an `aria-valuenow` label of 6!
A better way would be to store the value first, so something like this:
```
var itemValue = parseInt($(event.currentTarget).attr('data-id'));
var itemIndex = this.getIndexFromValue(itemValue);
```
Then do this:
```
this.setAltText(itemValue);
```
This also needs changing in `onDragReleased`

Clicking on the slider bar also results in an incorrect `aria-valuenow` value - but I can't quite figure out how to fix it (it's in `onSliderSelected`)